py/tmux_helper.py

```
@@ -640,10 +640,17 @@ def ensure_process(
     if not created_new:
         ensure_process("monitor", "btm", ["btm"])
 
-    # Mac-only windows
+    # Desktop Mac only - caffeinate to keep machine awake (skip on laptops)
     if is_mac:
-        ensure_process("awake", "caffeinate", ["caffeinate", "-d", "-i", "-s"])
+        model = subprocess.run(
+            ["sysctl", "-n", "hw.model"], capture_output=True, text=True
```

Ruff (0.14.10)

Starting a process with a partial executable path

(S607)